.travis.yml

```diff
@@ -1,7 +1,10 @@
 language: python
 python:
-    - "2.6"
     - "2.7"
+    - "3.2"
+    - "3.3"
+    - "3.4"
+    - "3.5"
 install:
-    - pip install -r requirements.txt --use-mirrors
+    - pip install virtualenv==13.1.2
```

Comment:
Yeah, it wasn't needed. Looks like travis is configured with virtualenv and pip version old enough to work find with 3.2.